cmd/fs-v1.go

```diff
@@ -203,7 +203,7 @@ func (fs fsObjects) statBucketDir(bucket string) (os.FileInfo, error) {
 
 // MakeBucket - create a new bucket, returns if it
 // already exists.
-func (fs fsObjects) MakeBucket(bucket string) error {
+func (fs fsObjects) MakeBucket(location, bucket string) error {
```

Comment:
Location should be the last argument .. All optional arguments go to the end.   Also this method signature change is not needed.